### ISSUE TITLE
Raise max password length

### ISSUE THREAD

**Issue by ChandonPierre** [cloudbase/cloudbase-init#114]
Maximum password length appears to be hardcoded to `20`:

https://github.com/cloudbase/cloudbase-init/blob/912eea88ed13fdb8abf62e4b455e19a18c21df81/cloudbaseinit/osutils/windows.py#L1697-L1698

Modern/currently supported versions of Windows [should be able to accept password lengths of 128 characters](https://support.microsoft.com/en-us/topic/minimum-password-length-auditing-and-enforcement-on-certain-versions-of-windows-5ef7fecf-3325-f56b-cc10-4fd565aacc59).

**Comment by daevski** [comment]
yeah, what is going on here??? This is not correct and needs to be updated! Thanks for creating this issue @ChandonPierre.

Workaround for now, set a 30 char password and then after build change it what you really need.
